### DISCUSSION TITLE
Setting ieee address in zzh ?

### DISCUSSION THREAD

**Discussion by tb205gti** [Koenkk/zigbee2mqtt#10320] [Devices]
I'm updating from older C2531 to a more powerfull Electrolama zig-a-zig-ah! (zzh!)

Now - to make things easier I can see it is recommended to write the ieee address from the old coordinator to the new one, but I'm unsure on how to do it as I cannot find any guides :)

Any hints?

**Comment by kirelam** [comment]
I have the same issue. I have a sonoff zigbee 3.0 and want to upgrade, but CCS  is not that easy and it is not that straight forward. Also i cannot find the right file/sdk to edit. I tried to play with flash programmer 2.0 to add the old IEEE adress, but that did not work out. I wished there was some dummy manual with some guidence in it, about the main editing and changing the IEEE.

**Comment by tb205gti** [comment]
I found it - using the -i flag should work. :)

**Reply by kirelam** [reply]
I what program and how? Please provider hints!!!

**Reply by Danit2** [reply]
Use this: 

https://github.com/Koenkk/zigbee2mqtt/discussions/9710
